src/ontology/mondo-edit.obo

```diff
@@ -428289,6 +428289,23 @@ synonym: "actin myopathy" EXACT []
 is_a: MONDO:0016139 {source="https://github.com/monarch-initiative/mondo/issues/900"} ! qualitative or quantitative protein defects in neuromuscular diseases
 is_a: MONDO:0019952 {source="https://github.com/monarch-initiative/mondo/issues/900"} ! congenital myopathy
 relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/129 ! ACTA1
+
+[Term]
+id: MONDO:0100085
+name: cancer of long bone of upper limb
+def: "A cancer that involves the upper limb long bone." []
+synonym: "cancer of fore limb long bone" EXACT []
+synonym: "cancer of long bone of fore limb" EXACT []
+synonym: "cancer of long bone of forelimb" EXACT []
+synonym: "cancer of long bone of upper extremity" EXACT []
+synonym: "cancer of upper extremity long bone" EXACT []
+synonym: "long bones of upper limb cancer" EXACT []
+synonym: "malignant neoplasm of upper limb long bone" EXACT []
+synonym: "malignant upper limb long bone neoplasm" EXACT []
+is_a: MONDO:0024311 ! cancer affecting bone of limb skeleton
+intersection_of: MONDO:0004992 ! cancer
+intersection_of: disease_has_location UBERON:0002102 ! forelimb
```

Comment:
you don't need this one here. 'forelimb long bone' is sufficient. And it's best to conform to the pattern in cancer.yaml which has one slop for relation

Reply:
got it - I fixed this